heuristic_baseline.py

```diff
@@ -0,0 +1,44 @@
+from dataloader import *
+from model import heuristic
+import numpy as np
+import torch
+from matplotlib import pyplot as plt
+
+
+class heuristic_run:
+    """
+    The class to run the heuristic model. No training needed.
+    Parameters can be tuned in model.heuristic
+    """
+    def __init__(self):
+        self.dataloader = single_dataloader()
+        self.num_data = len(self.dataloader)
+        self.model = heuristic()
```

Comment:
For a more general analysis class, should take the test set as an input with the model.